cmd/bundle/variables.go

```diff
@@ -6,4 +6,5 @@ import (
 
 func initVariableFlag(cmd *cobra.Command) {
 	cmd.PersistentFlags().StringSlice("var", []string{}, `set values for variables defined in bundle config. Example: --var="foo=bar"`)
+	cmd.PersistentFlags().String("vars-file-path", "", `file path to a JSON file containing variables. Example: --vars-file-path="/path/to/vars.json"`)
```

Comment:
Let's call this `--var-file`, shorter and same meaning.

Reply:
+1 or --varfile, don't have to remember to put dash.